DevEdu.API/DevEdu.Business/ValidationHelpers/CourseValidationHelper.cs

```diff
@@ -13,11 +14,20 @@ public CourseValidationHelper(ICourseRepository courseRepository)
             _courseRepository = courseRepository;
         }
 
-        public void CheckCourseExistence(int courseId)
+        public CourseDto CheckCourseExistence(int courseId)
```

Comment:
please rename to GetCourseByIdAndThrowIfNotFound

Reply:
+